docker-compose.yaml

```
@@ -30,6 +29,23 @@ services:
       - discovery.type=single-node
     volumes:
       - elasticsearch-data:/usr/share/elasticsearch/data
+    healthcheck:
+      test: [ "CMD-SHELL", "curl --output /dev/null --silent http://127.0.0.1:9200" ]
```

```suggestion
      test: [ "CMD-SHELL", "curl --output /dev/null --silent http://127.0.0.1:9200/_cat/health" ]
```

It is the [build-in healthcheck feature](https://www.elastic.co/guide/en/elasticsearch/reference/current/cat-health.html)